app/src/main/java/org/mozilla/fenix/ext/ProtonIcons.kt

```diff
@@ -0,0 +1,106 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@file:Suppress("TooManyFunctions")
+
+package org.mozilla.fenix.ext
+
+import org.mozilla.fenix.FeatureFlags
+import org.mozilla.fenix.R
+
+fun getAddonsDrawable(): Int {
+    return if (FeatureFlags.newIconSet) {
+        R.drawable.ic_proton_addons_extensions
+    } else {
+        R.drawable.ic_addons_extensions
+    }
+}
```

Comment:
This is nice that we are doing it in a separate file and that we can easily clean this up when `newIconSet` is default true.